CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# 0.49.0
+
+* Fixed: Whiskey build doesn't fail if Whiskey configuration file contains invalid YAML.
+* Fixed: Whiskey fails to run on AppVeyor's Visual Studio 2013 build image, i.e. on .NET 4.6.2 or earlier.
+* Fixed: installing Node.js fails if the build's output directory doesn't exist.
+* Fixed: when downloading Node.js, exceptions not related to downloading don't get shown to the user.
+* The `Pester3` task is obsolete and is no longer supported. Use the `Pester4` task instead.
+* Warnings written by `Import-Module` are now hidden.
+* Fixed: `Resolve-WhiskeyTaskPath` can return extra files if searching on a case-sensitive file system and the directory
+being searched has no upper or lower case letters.
+* Fixed: the `DotNet` command fails when running some commands under .NET 6.0 SDK because .NET 6.0 is stricter about
+validating parameters.
+* Added a `NoLog` parameter to the `DotNet` task to turn off logging.

```

Comment:
Lately I've been partial to breaking up release notes in subsections of features/improvements/bug fixes/misc. For example:

```
# 0.49.0

## Features

* Added a `NoLog` parameter to the `DotNet` task to turn off logging.

## Bug Fixes

* Fixed: Whiskey build doesn't fail if Whiskey configuration file contains invalid YAML.
* Fixed: Whiskey fails to run on AppVeyor's Visual Studio 2013 build image, i.e. on .NET 4.6.2 or earlier.
* Fixed: installing Node.js fails if the build's output directory doesn't exist.
* Fixed: when downloading Node.js, exceptions not related to downloading don't get shown to the user.
* Fixed: `Resolve-WhiskeyTaskPath` can return extra files if searching on a case-sensitive file system and the directory
  being searched has no upper or lower case letters.
* Fixed: the `DotNet` command fails when running some commands under .NET 6.0 SDK because .NET 6.0 is stricter about
  validating parameters.

## Misc

* The `Pester3` task is obsolete and is no longer supported. Use the `Pester4` task instead.
* Warnings written by `Import-Module` are now hidden.
```

Reply:
I'm using the section headings recommended by [changelog.md](changelog.md):

* Added
* Changed
* Deprecated
* Fixed
* Removed
* Security

Reply:
Even better 👍